src/databricks/labs/ucx/azure/credentials.py

```diff
@@ -288,19 +296,25 @@ def _create_storage_credentials_for_storage_accounts(self) -> list[StorageCreden
                 validation_results = self._storage_credential_manager.validate(storage_credential_info, url)
             except BadRequest:
                 logger.warning(f"Could not validate storage credential {storage_credential_info.name} for url {url}")
+            except (PermissionDenied, NotFound):
+                logger.error("Error creating all access connectors. Deleting incomplete ones.")
```

Comment:
This is an error on validating the access connectors, not creating them

Reply:
updated message